tests/unit/channel/secure_channel_test.py

```diff
@@ -0,0 +1,30 @@
+import grpc
+import pytest
+from mock import Mock, patch
+
+from pyzeebe import create_secure_channel
+from pyzeebe.channel.channel_options import get_channel_options
+
+
+class TestCreateSecureChannel:
+    @pytest.fixture(autouse=True)
+    def secure_channel_mock(self, aio_grpc_channel: grpc.aio.Channel):
+        with patch("grpc.aio.secure_channel", return_value=aio_grpc_channel) as mock:
+            yield mock
+
+    def test_returns_grpc_channel(self):
+        channel = create_secure_channel()
+
+        assert isinstance(channel, grpc.aio.Channel)
```

Comment:
Should test that host/port is being set correctly (can maybe do that in test_calls_using_default_grpc_options)

Should test overriding host/port.